openupgradelib/openupgrade.py

```diff
@@ -26,11 +26,16 @@
 from openerp import release, tools, SUPERUSER_ID
 from openerp.tools.yaml_import import yaml_import
 from openerp.osv import orm
-from openerp.tools.mail import plaintext2html
 from openerp.modules.registry import RegistryManager
 import openerp.osv.fields
-import openerp.fields
 from . import openupgrade_tools
+if release >= "7.0":
+    from openerp.tools.mail import plaintext2html
+if release >= "8.0":
+    from openerp.fields.Many2many import Many2many, One2many
```

Comment:
I'm sorry to be picky, but isn't there a version_info in release that you could use? Like `release.version_info[0] == 7` or something like that?